velox/exec/HashTable.h

```diff
@@ -477,6 +477,9 @@ class HashTable : public BaseHashTable {
       uint64_t* FOLLY_NULLABLE hashes,
       int32_t numGroups);
 
+  /// Checks if we can parallelize the table build process for hash join.
+  bool applyParallelJoinBuild() const;
```

Comment:
Nit: the name of this method suggests that it will actually perform the build. Would it make sense to rename to supportsParallelJoinBuild?

Reply:
I see. How about rename it to canApplyParallelJoinBuild which apply is better than support here? Thanks!

Reply:
Sure.